coala_quickstart/info_extractors/PackageJSONInfoExtractor.py

```diff
@@ -0,0 +1,45 @@
+import json
+
+from coala_quickstart.info_extraction.InfoExtractor import InfoExtractor
+from coala_quickstart.info_extraction.Information import (
+    LicenseUsedInfo, ProjectDependencyInfo, IncludePathsInfo, ManPathsInfo,
+    VersionInfo)
+
+
+class PackageJSONInfoExtractor(InfoExtractor):
```

Comment:
Could you include a link to something like a RFC/spec defining what goes into a package.json file? Same thing for all other files too. It'd be neat to have a reference link right in the code.